chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+about all units that could appear in the data
+and their relationship to the research design,
+like sampling and treatment status once those are determined.
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this information,
+will save a ton of time later on,
+and will increase the quality of your research.
+
+You should create a master dataset
+for each unit of observation
+important to the research.
+A unit used for any significant research activity,
+like data collection or data analysis, 
+must always be considered an important unit of observation.
+Therefore, any unit
+that will be used in sampling or treatment assignment,
+must have a master dataset,
+and that master dataset -- not field data --
+should be used when sampling or assigning treatment.
+
+You also need to record how all datasets for each unit of observation
+will link or merge with each other as needed.
+This linking scheme is called a \textbf{data map}\sidenote{
+	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}.
+\index{data maps} 
+A data map is more than just a list of datasets.
+Its purpose is to specify the characteristics and linkages of those datasets.
+To link properly, the master datasets must include fully unique ID variables.\sidenote{
+	\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
+The master datasets should indicate whether datasets should be merge one-to-one,
+for example, merging baseline data and endline data that use the same unit of observation,
+or whether two datasets should be merged many-to-one, 
+for example, school administrative data merged with student data.
+Your data map must indicate which ID variables can be used and how to merge datasets.
+It is common that administrative data use IDs 
+that are different than the project IDs, 
+and the linkage between those should be clear from your master dataset.
+
+The data map should also include metadata about the handling of all information.
+These characteristics may be updated as the project progresses.
+For example, you will need to note the original source of each dataset,
+as well as the project folder where 
+the raw original data and codebooks are stored
+and where the back-ups for the each raw dataset are stored.
+
+Some of the characteristics in your master datasets and your data map
+should be filled in during the planning stage,
+but both of them should be active resources 
+that are updated as your project evolves.
+Finally, your master data should not include any unlabeled missing values. 
+If the information is missing for one unit, 
+then the reason should always be indicated with a code.
+An example for such reason could be that a unit was not included in the treatment assignment
+as it was not sampled in the first place,
+or was not located in the data collection at a given round.
+
+\subsection{Defining study comparisons using master data}
```

Comment:
the content below doesn't really match this heading yet (but the premise of the heading is great and gives a nice logical structure to the chapter). i think there needs to be more specific reference, something along the lines of now that the master data is set up you can do the treatment assignment. and making it clear that master data always needs the variables that describe the different research designs described. possibly even being explicit about what those variables are.

Reply:
moved to https://github.com/worldbank/dime-data-handbook/issues/468